### ISSUE TITLE
The Legend of Zelda Breath of the Wild not recognizing some amiibos

### ISSUE THREAD

**Issue by redkaio** [FuryBaguette/AmiiSwap#19]
Hey there. The new version of Amiiswap is wonderful. I could use it with Mario Kart and Odyssey with no issues so far. However, TLoZ Breath of the Wild is having trouble when I use specific amiibos. I tried so far most of them and the 30th Anniversary Ocarina of Time Link, 8 Bit Link, Archer Link and Majora's Mask Link are not working as intended. It seems that the game recognize them as the Toon Link amiibo. I'm assuming that because the content of the chests and the items generated when the amiibo is read match with the Toon Link features. I can provide videos, images and anything else that can be of help. Also, I tried different bin files of the amiibos I getting issues with. Thanks in advance!

**Comment by redkaio** [comment]
Hey there. The new version of Amiiswap is wonderful. I could use it with Mario Kart and Odyssey with no issues so far. However, TLoZ Breath of the Wild is having trouble when I use specific amiibos. I tried so far most of them and the 30th Anniversary Ocarina of Time Link, 8 Bit Link, Archer Link and Majora's Mask Link are not working as intended. It seems that the game recognize them as the Toon Link amiibo. I'm assuming that because the content of the chests and the items generated when the amiibo is read match with the Toon Link features. I can provide videos, images and anything else that can be of help. Also, I tried different bin files of the amiibos I getting issues with. Thanks in advance!
